python/paddle/fluid/backward.py

```diff
@@ -1187,8 +1227,14 @@ def _append_backward_ops_(block,
             grad_op_descs.extend(grad_op_desc)
             grad_to_var.update(op_grad_to_var)
 
+    # record mapping bewteen grad var name and var name (Only for auto parallel)
+    grad_var_to_var = None
+    if distop_context is not None:
+        grad_var_to_var = distop_context.grad_var_to_var[
```

Comment:
rename to `dist_op_context.grad_var_to_var_maps` ?

Reply:
it will be updated in the next pr.